niworkflows/__init__.py

```diff
@@ -23,3 +23,6 @@
     matplotlib.use("Agg")
 except ImportError:
     pass
+
+from . import _version
+__version__ = _version.get_versions()['version']
```

Comment:
I believe we put these lines in `__about__`. Maybe check that they're updated to this form?

```suggestion
```